src/harness/testcases/output/test_WINNF_FT_S_PCR_1/sas_uut_claimed_ppa_boundary.json

```diff
@@ -0,0 +1,1686 @@
+{
```

Comment:
This probably shouldn't be checked in since it will be different for each SAS.

Reply:
This   file is no longer required to checkin based on the reason mentioned     so we have removed this file.